src/test/suite/editors/playgroundController.test.ts

```diff
@@ -193,41 +192,50 @@ suite('Playground Controller Test Suite', function () {
     testPlaygroundController._activeTextEditor = undefined;
 
     test('run all playground blocks should throw the playground not found error', async () => {
-      const errorMessage =
-        "Please open a '.mongodb' playground file before running it.";
-
-      fakeShowErrorMessage.resolves(errorMessage);
+      const expectedMessage = "Please open a '.mongodb' playground file before running it.";
+      const fakeShowErrorMessage: any = sinon.fake();
```

Comment:
We can probably use the `sinon.SinonFake` type here

Reply:
We took a look at this before and kept any because `Property 'firstArg' does not exist on type 'SinonFake'.`.